internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -543,7 +543,7 @@ func resourceDatabricksWorkspaceRead(d *pluginsdk.ResourceData, meta interface{}
 		}
 		d.Set("managed_resource_group_id", props.ManagedResourceGroupID)
 		d.Set("managed_resource_group_name", managedResourceGroupID.ResourceGroup)
-		d.Set("public_network_access_enabled", (props.PublicNetworkAccess == databricks.PublicNetworkAccessEnabled))
+		d.Set("public_network_access_enabled", (props.PublicNetworkAccess != databricks.PublicNetworkAccessDisabled))
```

Comment:
I think you can ditch the outer wrapper?

```suggestion
		d.Set("public_network_access_enabled", props.PublicNetworkAccess != databricks.PublicNetworkAccessDisabled)
```

Reply:
Hmm, probably? 🤷🏻